ui/src/shared/components/AutoRefresh.tsx

```diff
@@ -231,6 +232,20 @@ const AutoRefresh = (
         _.get(resp, 'results', []).every(r => Object.keys(r).length > 1)
       )
     }
+
+    private isActiveTemplatesDifferent(prevProps: Props) {
+      const prevTemplates = this.getActiveTemplates(prevProps.templates)
+      const templates = this.getActiveTemplates(this.props.templates)
+
+      return !_.isEqual(prevTemplates, templates)
+    }
+
+    private getActiveTemplates(tempVars: Template[]): Template[] {
+      return _.map(tempVars, tempVar => ({
+        ...tempVar,
+        values: tempVar.values.filter(v => v.selected || v.localSelected),
```

Comment:
ooh! I think this should look at `v.localSelected` first, and then to `v.selected` !!